File: app/src/main/java/com/slapshotapps/dragonshockey/DragonsApplication.java
package com.slapshotapps.dragonshockey;

import android.app.Application;

import timber.log.Timber;

/**
 * Created by willmetz on 5/30/16.
 */

public class DragonsApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: app/src/main/java/com/slapshotapps/dragonshockey/DragonsApplication.kt
package com.slapshotapps.dragonshockey

import android.app.Application
import com.slapshotapps.dragonshockey.utils.NotificationUtil
import timber.log.Timber


/**
 * The [Application] class for the app.
 */
class DragonsApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }

        NotificationUtil.createNotificationChannel(this)
    }


}
